sdk/communication/communication-chat/test/internal/chatClient.mocked.spec.ts

```diff
@@ -394,4 +394,36 @@ describe("[Mocked] ChatClient", async () => {
       }
     },
   );
+
```

Comment:
Please check if the tests are pass

Reply:
They pass.

Reply:
I need to update the recordings in the test tool.